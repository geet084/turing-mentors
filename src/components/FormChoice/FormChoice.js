import React from 'react';
import { Link } from 'react-router-dom';

export const FormChoice = () => {
  return (
    <div className="form-choice">
      <Link to='/mentee'>Mentee</Link>
      <Link to='/mentor'>Mentor</Link>
    </div>
  )
}

export default FormChoice;