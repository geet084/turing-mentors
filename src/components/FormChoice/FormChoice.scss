.form-choice {
  display: flex;
  flex-direction: column;
  justify-content: center;
  padding: 30px 0;
  width: 100%;
  div {
    margin: 20px auto;
  }
  a {
    text-decoration: none;
    color: grey;
    font-size: 2.25rem;
    padding: 2px 4px;
    border: 1px solid white;
    border-radius: 10px;
    box-shadow: 4px 4px 8px -2px rgba(0,0,0,0.25);
    &:hover {
      cursor: pointer;
      margin-bottom: -2px;
      border-bottom: 2px solid #12f1fc;
    }
  }
}

.form-link {
  margin: 0 15px;
}