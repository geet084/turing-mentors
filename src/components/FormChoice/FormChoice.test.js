import React from 'react';
import { shallow } from 'enzyme';
import { FormChoice } from './FormChoice';

describe('FormChoice', () => {
  let wrapper;
  it('should have match the correct snapshot', () => {
    wrapper = shallow(<FormChoice />);
    expect(wrapper).toMatchSnapshot();
  });
});