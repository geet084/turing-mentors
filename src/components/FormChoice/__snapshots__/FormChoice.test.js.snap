// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormChoice should have match the correct snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <FormChoice />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Link
          className="form-link"
          to="/mentee"
        >
          Mentee
        </Link>,
        <Link
          className="form-link"
          to="/mentor"
        >
          Mentor
        </Link>,
      ],
      "className": "form-choice",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": "Mentee",
          "className": "form-link",
          "to": "/mentee",
        },
        "ref": null,
        "rendered": "Mentee",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": "Mentor",
          "className": "form-link",
          "to": "/mentor",
        },
        "ref": null,
        "rendered": "Mentor",
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Link
            className="form-link"
            to="/mentee"
          >
            Mentee
          </Link>,
          <Link
            className="form-link"
            to="/mentor"
          >
            Mentor
          </Link>,
        ],
        "className": "form-choice",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "Mentee",
            "className": "form-link",
            "to": "/mentee",
          },
          "ref": null,
          "rendered": "Mentee",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "Mentor",
            "className": "form-link",
            "to": "/mentor",
          },
          "ref": null,
          "rendered": "Mentor",
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
